docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -1,7 +1,9 @@
 [id='{context}-con-workflow-step']
-= Building Your First Workflow Step by Reusing a Template
+= Building Your First Workflow Step
 
-A basic knowledge of Angular.js is required to build a *Workflow Step* in {PROJECT_NAME}.
+Steps are internal part of the {PROJECT_NAME} ecosystem.
```

Comment:
```Steps are a internal part of {PROJECT_NAME}```

Reply:
You do not like ecosystem? :) Yeah removing that:)